### PR TITLE
Fix make symfony security bundle optional

### PR DESCRIPTION
**Pull request by cedvan**
Just fix make symfony security bundle optional

### CODE REVIEW

File: src/M6Web/Bundle/LogBridgeBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@ services:
             - %m6web_log_bridge.ignore_headers%
             - %m6web_log_bridge.prefix_key%
         calls:
-            - [ setTokenStorage, [ !@security.token_storage ] ]
\ No newline at end of file
+            - [ setTokenStorage, [ @?security.token_storage ] ]
```
Comment (mikaelrandy) [inline src/M6Web/Bundle/LogBridgeBundle/Resources/config/services.yml]: Missing newline

Comment (cedvan) [inline src/M6Web/Bundle/LogBridgeBundle/Resources/config/services.yml]: done

**Comment by mojoLyon** [comment]
:metal:

**Comment by cedvan** [comment]
New release [3.0.1](https://github.com/M6Web/LogBridgeBundle/releases/tag/v3.0.1)

**Pull request merged**